README.md

```diff
@@ -40,32 +40,32 @@
  On this page display the names of files & folders (as determined by an API call) for the selected usernames’ repository.
  
  ## Task 7.
-  Stretch exercise.
+  //Stretch exercise.
   
-  Make a new route GET /github/users/:username/repos/:repoName/*path
+  //Make a new route GET /github/users/:username/repos/:repoName/*path
   The path will be path to file I.E app/controllers/HelloWorldController.scala, you can base64 encode path for ease
   If the path given resolves to a file, display the plaintext of the file on the page.
   If the path given is a folder, display the files & folder names (In the same way it's displayed in the first part of Task 4)
   
 ## Task 8. 
- (Assuming task 7 stretch is complete)
+ //(Assuming task 7 stretch is complete)
```

Comment:
Probably not needed as all these tasks look like you can't move on without doing the one prior